lte/gateway/c/connection_tracker/EventTracker.h

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <stdio.h>
```

Comment:
I'd recommend to only include files that the header will need, as minimum as possible, and then include all necessary header files on the source .cpp file, let's clean this up with only the necessary include files for the header file and move all of the other to the EventTracker.cpp file